app/components/invites/steps/SelectFeedsStep.module.css

```
@@ -30,7 +40,7 @@
 }
 
 @media(--tablet) {
```

Biome (2.3.14)

[error] Don't use unknown media feature names.

Unexpected unknown media feature name.
You should use media feature names defined in the CSS Specifications.

(lint/correctness/noUnknownMediaFeatureName)